### PR TITLE
First working acceptance test for appointments/search path; ensures only the correct appointment is displayed based on its start date

### LINKED ISSUE #689: Build acceptance test for appointments.search
In order to effectively use the no missing translations test, we need to make sure that every page is visited during acceptance tests.

This issue tracks the missing appointments.search acceptance test

for ember new commers the value some.route is the route that's missing you can find them under router.js

### PR DESCRIPTION
**Pull request by richiethomas**
Addresses #689 

**Changes proposed in this pull request:**

- This PR adds acceptance test for appointment searches based on start date
I plan to write more tests for the other searchable appointment attributes.  Submitting this now for initial feedback.

cc @HospitalRun/core-maintainers

### CODE REVIEW

File: tests/acceptance/appointments-test.js [hunk 1]
```diff
@@ -201,6 +201,50 @@ test('Appointment calendar', function(assert) {
...
+      // debugger;
+      let desiredDate = moment().endOf('day').add(363, 'days').format('l');
+      let datePicker = '.test-selected-start-date input';
+      selectDate(datePicker, desiredDate);
+      click('button:contains(Search)');
+    });
+
+    andThen(function() {
+      let date = moment().endOf('day').add(1, 'years').add(2, 'days').format('l');
+      let element = 'tr:contains(' + date + ')';
```
Comment (jkleinsc) [inline tests/acceptance/appointments-test.js]: This line is failing ESLint.
See http://eslint.org/docs/rules/prefer-template for details on how to change.

File: tests/acceptance/appointments-test.js [hunk 2]
```diff
@@ -201,6 +201,50 @@ test('Appointment calendar', function(assert) {
...
+      findWithAssert(element);
+      date = moment().startOf('day').add(1, 'years');
+      element = find('tr:contains(' + date + ')');
```
Comment (jkleinsc) [inline tests/acceptance/appointments-test.js]: This line is also failing ESLint.
See http://eslint.org/docs/rules/prefer-template for details on how to change.

File: tests/acceptance/appointments-test.js (hunk 1 above)
Comment (richiethomas) [inline tests/acceptance/appointments-test.js]: Fixed in latest PR.

File: tests/acceptance/appointments-test.js (hunk 2 above)
Comment (richiethomas) [inline tests/acceptance/appointments-test.js]: Fixed in latest PR.

**Comment by richiethomas** [comment]
@jkleinsc fixed those ESLint errors.

**Comment by jkleinsc** [comment]
Looks good to me @richiethomas.  Thanks for working on this PR!

**Pull request merged**